LDAR_Sim/src/default_parameters/m_default.yml

```diff
@@ -49,5 +49,6 @@ follow_up:
   min_followup_days_to_end: 0 # days
   proportion: 1.0 # 0 to 1.0
   redundancy_filter: "recent" # recent/max/average
+  sort: True # True/False
```

Comment:
I wonder if a name like prioritize by rate might be better than sort here. While sort is what we are doing it might not be clear to external users what that means.

Reply:
Renamed to sort_by_rate and User manual was updated.